src/tests/calculator.test.js

```diff
@@ -0,0 +1,9 @@
+import renderer from 'react-test-renderer';
+import { Calculator } from '../components';
+
+it('renders calculator page correctly', () => {
+  const tree = renderer
+    .create(<Calculator />)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] You are missing tests for simulating user interactions. Kindly add user interaction tests using the React Testing Library.

  If you are unfamiliar with it you can visit [this tutorial](https://www.robinwieruch.de/react-testing-library) on how to add tests for that.